docs/src/main/sphinx/admin/properties-query-management.rst

```diff
@@ -93,3 +93,11 @@ removed based on age.
 The minimal age of a query in the history before it is expired. An expired
 query is removed from the query history buffer and no longer available in
 the :doc:`/admin/web-interface`.
+
+``query.max-cpu-time``
+^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-duration`
+* **Default value:** ``1_000_000_000d``
+
+Limits CPU time used by a query.
```

Comment:
What is the result of applying this property?

Reply:
Queries that use more CPU time are cancelled from all I know.